packages/viewer/src/scenes/raycaster.ts

```diff
@@ -17,7 +17,9 @@ export class Raycaster {
    */
   public async hitItems(
     point: Point.Point
-  ): Promise<vertexvis.protobuf.stream.IStreamResponse> {
-    return this.stream.hitItems(UUID.create(), { point });
+  ): Promise<vertexvis.protobuf.stream.IHitItemsResult> {
+    return this.stream
+      .hitItems(UUID.create(), { point })
```

Comment:
Maybe it'd be better for `stream-api` to assume that you want a response and have the package generate the request ID for you, instead of passing it in. We could use a boolean as the last argument to indicate this is a fire-and-forget type request, which won't supply a request ID. What do you think?